Sources/EasyNotificationBadge/EasyNotificationBadge.swift

```diff
@@ -229,6 +233,7 @@ public struct BadgeAppearance {
     public var duration: TimeInterval
     public var distanceFromCenterY: CGFloat
     public var distanceFromCenterX: CGFloat
+    public var radius: CGFloat
```

Comment:
This should be optional.
```suggestion
    public var radius: CGFloat?
```